explanation/intro-to/backups.md

```diff
@@ -1,36 +1,36 @@
+The easiest way to create backups is to use dedicated tools like [Bacula](http://www.bacula.org/) or [rsnapshot](https://rsnapshot.org/). These tools have easy-to-use interface or CLI to help simplify the backup management process. They have powerful features such as automation, compression, data recovery, encryption and incremental backups.
 
 * **Bacula**
-  This tool uses incremental backups, which only store changes made since the last backup. This can significantly decrease the storage space and backup time required. It can also manage backups of multiple systems over a network. With more advanced features and support for additional customisation, it is often used by users with more complex needs (e.g. in enterprise environments).
+  Bacula's advanced features and support for additional customization make it a good choice for enterprise systems or users with complex needs. Bacula can manage backups of multiple systems over a network. Bacula uses incremental backups which only store changes made since the last backup. This can significantly decrease storage space needs and backup time. 
 
-  * Find out {ref}`how to install and configure Bacula <install-bacula>`.
+    * Find out {ref}`how to install and configure Bacula <install-bacula>`.
   
-* **rsnapshot** uses rsync to take periodic "snapshots" of your files, which makes it easier to access previous versions. It's often used for local backups on a single system and is ideal for individual users or small-scale organisations who want a simpler and more efficient solution.
+* **rsnapshot** is ideal for individual users or small-scale organizations who want a simple and efficient solution. **rsnapshot** uses `rsync` to take periodic "snapshots" of your files so it is easy to access earlier versions. It's often used for local backups on a single system.
 
   * Find out {ref}`how to install and configure rsnapshot <install-rsnapshot>`.
 
 ### Shell scripts
 
-Using shell scripts to manage your backups can be easy or complicated, depending on the complexity of your setup. However, the advantage of shell scripts over using backup utilities, is that they offer full flexibility and customisation. Through backup shell scripts, you can fully tailor the backup process to your specific requirements without using third-party tools.
+You can tailor the backup process to your specific requirements with shell scripts. The advantage of shell scripts over backup utilities is they offer full flexibility and customization. 
 
 Refer to this guide for instructions on {ref}`how to use shell scripts for backups <back-up-using-shell-scripts>` -- or you can take a look at our reference examples:
-* {ref}`A basic backup shell script <basic-backup-shell-script>`
-* {ref}`An example of archive rotation with shell scripts <archive-rotation-shell-script>`
+  * {ref}`A basic backup shell script <basic-backup-shell-script>`
+  * {ref}`An example of archive rotation with shell scripts <archive-rotation-shell-script>`
```

Comment:
There's actually no need to add indentation here, it renders exactly the same on the page